go.mod

```diff
@@ -62,7 +62,6 @@ require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/eapache/go-resiliency v1.2.0
 	github.com/eclipse/paho.mqtt.golang v1.3.5
-	github.com/elastic/ecs v1.12.0
```

Comment:
Are we completely removing it as a dependency? From my understanding I think we are, just making sure.